cruise.umple/test/cruise/umple/compiler/UmpleParserTest.java

```diff
@@ -2560,6 +2560,70 @@ public void methodNameConflictUsingDefaulted()
     assertFailedParse("045_phpDefaultedNameConflict.ump", 48);
     assertFailedParse("045_rubyDefaultedNameConflict.ump", 48);
   }
+  
+  @Test
+  public void autoGeneratedAPIMethods()
```

Comment:
It is better to keep test cases simple.  Each test case should be dealing with a specific case. It helps later to explore the test case easier.  One test case for every stereotype. You can keep the source model the same for all test cases.